docs/source/user/context.rst

```diff
@@ -0,0 +1,20 @@
+Context
+=======
+
+An LLM is only as good as its context. Therefore we provide a number of ways to manually manage and inspect context as well as a few tools to attempt to automatically select a context.
```

Comment:
"Context refers to portions of your code/files which are sent to the LLM along with your question/task, adn an LLM is only as good as its context."

Anchoring the word 'context' up-front might be helpful for some